modin/experimental/pandas/test/test_io_exp.py

```diff
@@ -159,8 +159,8 @@ def test_read_single_csv_with_parse_dates(self, parse_dates):
                 modin_df = pd.read_csv_glob(
                     time_parsing_csv_path, parse_dates=parse_dates
                 )
-                # Call __repr__ on the modin df to force it to materialize.
-                repr(modin_df)
+                # Call _to_pandas on the modin df to force it to materialize.
+                _ = modin_df._to_pandas()
```

Comment:
Maybe we should use public `utils.to_pandas()` function instead? The protected (name starting with `_`) is called so for a reason, and is subject to (potential) change.